manager/testdata/notebook/read-flow/setup.sh

```diff
@@ -11,7 +12,12 @@ kubectl -n fybrik-notebook-sample apply -f s3credentials.yaml
 
 # Avoid using webhooks in tests
 kubectl delete validatingwebhookconfiguration fybrik-system-validating-webhook
-# Use master version of arrow-flight-module according to https://github.com/fybrik/arrow-flight-module#version-compatbility-matrix
-kubectl apply -f https://raw.githubusercontent.com/fybrik/arrow-flight-module/master/module.yaml -n fybrik-system
+
+if [[ -z "${OLD_AFM_VERSION}" ]]; then
+  # Use master version of arrow-flight-module according to https://github.com/fybrik/arrow-flight-module#version-compatbility-matrix
+  kubectl apply -f https://raw.githubusercontent.com/fybrik/arrow-flight-module/master/module.yaml -n fybrik-system
+else
+    kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/${OLD_AFM_VERSION}/module.yaml -n fybrik-system
```

Comment:
indentation is slightly different than the upper line.

Reply:
fixed